modules/_canvas-kit-react/index.ts

```diff
@@ -9,6 +9,7 @@ export * from '@workday/canvas-kit-react-checkbox';
 export * from '@workday/canvas-kit-react-color-picker';
 export * from '@workday/canvas-kit-react-common';
 export * from '@workday/canvas-kit-react-cookie-banner';
+export * from '@workday/canvas-kit-react-expandable';
```

Comment:
I think all new components are going through Labs first. @alanbsmith is currently working on documentation for the process of a new component.

Reply:
Sounds good, I'll move it there.